app/forms/collection_form.rb

```diff
@@ -41,7 +41,10 @@ def build_child
 
   def save
     if valid?
-      run_callbacks(:save) { collection.all?(&:save) }
+      run_callbacks(:save) do
```

Comment:
I think it makes sense to say its all or nothing however this is only a problem if the app has some sort of DB issue.

Speaking of DB problems.. http://www.postgresql.org/docs/9.4/static/runtime-config-autovacuum.html is something Ive seen cause issues before but I don't think we'll be doing enough traffic for this to be of a concern.